### PR TITLE
Add node.js capabilities, exporting features, update README

### PR DESCRIPTION
**Pull request by gendelbendel**
I was interested in using this project and thought it would be valuable for others using Javascript to be able to pull in this data via npm.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,43 @@
...
+## Using in your project
+
+Simply `require` this module.
+
+```
+// example.js
+const myData = require('idleon-data');
+```
+
+Each `.json` file will have its own key based on the filename. e.g. `/maps/itemNames.json` will be accessable at the `itemNames` key.
```
Comment (Corbeno) [inline README.md:32]: It might be worth showing a quick example here to get the item name of some random object to show how easy it is

**Review by Corbeno — COMMENTED** [review]
Thank you for contributing to this! This is a great addition and will allow for even more people to use this!

Other then the minor addition to the readme, this looks great! I will have to try it out myself later tonight.

File: README.md (hunk 1 above)
Comment (gendelbendel) [reply README.md:32]: Will update the PR at some point today (probably after work) to include this.

**Comment by gendelbendel** [comment]
Let me know if there are other examples or changes you are desiring!

**Comment by desophos** [comment]
This is fantastic, thank you for doing this! Unfortunately, I'm not a JS dev so I don't have much else to say here. I think this one's up to Corbeno.

**Comment by Corbeno** [comment]
Sorry I wasn't able to address it this week. I have been swamped with work/classes.

This all looks good to me though. Thank you for your work!

**Pull request merged**